src/sagemaker/estimator.py

```diff
@@ -3414,6 +3414,7 @@ def __init__(
         self.checkpoint_s3_uri = checkpoint_s3_uri
         self.checkpoint_local_path = checkpoint_local_path
         self.enable_sagemaker_metrics = enable_sagemaker_metrics
+        self.unsupported_dlc_image_for_sm_parallelism = ["2.0.1-gpu-py310-cu121-ubuntu20.04-sagemaker-pr-3303"]
```

Comment:
this is not correct one and has tag for image built in PR `pr-3033`

do you want to make it generic for future images or want to add image tags explicitly?

Reply:
What do you mean this is not the correct one? This is the tag for the CUDA 12 p5 image correct? And yes, we want to blacklist images explicitly rather than having a generic blocker for now.

Reply:
that image is built on PR with tag `2.0.1-gpu-py310-cu121-ubuntu20.04-sagemaker-pr-330`, prod one will have generic tag like `2.0.1-gpu-py310-cu121-ubuntu20.04-sagemaker` and also prod ones have more additional tags as well like `2.0.1-gpu-py310-cu121-ubuntu20.04-sagemaker-v2` . Check examples here https://github.com/aws/deep-learning-containers/releases